keyboards/ai03/equinox_xl/info.json

```diff
@@ -0,0 +1,89 @@
+{
+    "manufacturer": "ai03 Design Studio",
+    "keyboard_name": "Equinox XL",
+    "maintainer": "ai03",
+    "bootloader": "atmel-dfu",
+    "build": {
+        "debounce_type": "asym_eager_defer_pk"
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "caps_lock": "F1",
+        "on_state": 0
+    },
+    "matrix_pins": {
+        "cols": ["F5", "C7", "C6", "B6", "B5", "B4", "D7", "D6", "D4", "D5", "D3", "D2", "D1", "D0"],
+        "rows": ["B7", "F4", "F6", "F7"]
+    },
+    "processor": "atmega32u4",
+    "url": "https://ai03.com/projects/equinox-xl/",
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x0020",
+        "vid": "0xA103"
+    },
+    "layouts": {
```

Comment:
could matrix co-ordinates (e.g. `1,12`) for the following keys be confirmed?:
- ISO Hash
- ISO Enter
- 2.25u Left Shift
- 2.75u Right Shift
- 7u Spacebar

Reply:
I apologize for the delay; ended up busy with things that came up in life.

I've double-checked the requested keys:
- ISO hash - common matrix position with ANSI pipe (0, 13)
- ISO enter - common matrix position with ANSI enter (1, 13)
- 2.25U left shift - common matrix position with 1.25U left shift (2, 0)
- 2.75U right shift - common matrix position with the 1U to the right of 1.75U right shift (2, 13)
- 7U spacebar - common matrix position with the 1U central key of split space (3, 7)